substrate/frame/contracts/fixtures/contracts/return_with_data.rs

```diff
@@ -38,6 +38,9 @@ pub extern "C" fn call() {
 		output: [u8],
 	);
 
+	// burn some PoV
```

Comment:
Nit: comment could include some of below discussion for context

Reply:
Yep